editor/src/clj/internal/node.clj

```diff
@@ -1727,11 +1748,23 @@
                   ;; produced property to set the :original-value here.
                   props-with-overrides-and-original-values
                   (reduce-kv (fn [props prop-kw orig-prop]
-                               (let [original-value (:value orig-prop)]
-                                 (if (or (contains? properties prop-kw)
-                                         (and (not (declared? prop-kw))
-                                              (get-in props [prop-kw :assoc-original-value?])))
-                                   (update props prop-kw assoc :original-value original-value)
+                               (let [prop (get props prop-kw)
+
+                                     assoc-original-value
+                                     (cond
+                                       (nil? prop)
+                                       false
+
+                                       (contains? properties prop-kw)
+                                       (:assoc-original-value? prop true)
+
+                                       (not (declared? prop-kw))
+                                       (:assoc-original-value? prop false)
+
+                                       :else
+                                       false)]
```

Comment:
We do some trickery in the `_properties` output of `GuiNode` to make properties appear overridden based on what `current-layout` is selected for editing. For our purposes, we need to stop this code from `associng` in the `:original-value` from the original node. You can now set `:assoc-original-value?` to `false` in the produced property to disable this. Previously you could only set it to `true` to force it to happen.